### ISSUE TITLE
Problem downloading the test runner

### ISSUE THREAD

**Issue by aneksamun** [zio/zio-intellij#408]
The following error(s) occurred while downloading the ZIO Test runner files:

```
Download error: zio-test-intellij_2.13:1.0.18"
Cause:
org.jetbrains.plugins.scala.DependencyManagerBase$ResolveException: unresolved dependency: dev.zio#zio-test-intellij_2.13;1.0.18: not found
```
### Additional information:
<details>

ZIO plugin version: 2022.3.20.0
Scala plugin version: 2022.3.18
ZIO version(s): 1.0.18, 1.0.17
Scala version(s): 2.13.10, 2.13.10
sbt version: 1.7.2
IntelliJ version: IntelliJ IDEA 2022.3.2
Build #IU-223.8617.56
Java version: 17.0.5+1-b653.25
Operating System: x86_64 Mac OS X (13.2.1, x86_64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o.
</details>

**Comment by aneksamun** [comment]
It seems that I have an older version than in other two issues. Not sure whether that matters, thought.

**Comment by hmemcpy** [comment]
Aah oops! I didn't realize there is a new version of ZIO 1.0 out! I'll have to release a runner for it. Thanks for reporting, I'll fix it!

**Comment by aneksamun** [comment]
Thank you!

**Comment by hmemcpy** [comment]
Thanks again for the heads-up! Just published a 1.0.18-compatible runner (will take about 10 minutes for it to appear on maven central). Please reload the project and the download link on the popup should work now!

**Comment by aneksamun** [comment]
It all works great! Thanks for effort
